src/StudyEntities/Candidate/SessionID.php

```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+/**
+ * This defines what a SessionID should be so it can be passed as a typed
+ * parameter in functions.
+ *
+ * PHP Version 7
+ *
+ * @category StudyEntities
+ * @package  LORIS
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate;
+
+/**
+ * A representation of a SessionID object. A SessionID is always an integer with a
+ * maximum of 10 digits (as definied in the database `session` table.
+ *
+ * @category StudyEntities
+ * @package  LORIS
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class SessionID extends ValidatableIdentifier
```

Comment:
This is in  `namespace LORIS\StudyEntities\Candidate;` A SessionID is not a validatable candidate identifier.

Reply:
@driusan I have no preference about where the files are stored but I would like to use the ValidatableIdentifier class. It seems generic enough to be used for classes beyond CandID|PSCID|ExternalID.

How do you feel about me moving Identifier and ValidatableIdentifier to php/libraries and updating the `StudyEntities\Candidate` classes accordingly? SessionID can also be moved to libraries as you're right it doesn't belong in the Candidate folder.

Reply:
I think it would be a good idea to keep the StudyEntities namespace self-contained enough that it has no coupling to LORIS and could eventually be moved into a different repo and just composer required by LORIS (whether or not that ever happens, it should be possible to do it since a bunch of models for scientific study purposes is useful outside of LORIS) and I think it's a good idea to cleanly separate whether an identifier is a Candidate identifier, or an identifier for something else in the class hierarchy.

How about just making a copy of ValidatableIdentifier in php/libraries?

Reply:
@xlecours @ridz1208

Reply:
Sure I can make a copy. Happy to go with the flow~

Reply:
The files are now in the libraries/ folder